src/components/attendee/Input/Input.module.scss

```diff
@@ -26,3 +35,25 @@
 .border {
   border: border.$radius--s3 solid theme.$sparkle--border;
 }
+
+.invalid {
+  @include error.input-error;
+}
+
+.errorIcon {
+  background-image: url(/assets/icons/input-error-icon.svg);
+  height: 18px;
+  width: 18px;
+  margin-left: -24px;
```

Comment:
[NOTE] Oddly specific values. I'm guessing that's the icon's size, but do we have standardized icon sizes? 

A thing to improve the third line: a comment explaining just why `-24px` should be helpful. 

If there is a relation to some other value, maybe it should be explained or tied more closely (like if one value needs to be `$-x` and another `2 * $-x` )

Reply:
I've added these icon sizes to `scss` constants along with others
as for the margin thing, it's just used to move this icon in the `flex` container inside the `input` itself
I've added a comment just in case
[a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)